### PR TITLE
Introduce new volume type and objectstorage endpoint

### PR DESCRIPTION
**Pull request by lgrebe-sys**
os-13135

### CODE REVIEW

File: user/pages/04.Reference/05.object-storage/docs.en.md [hunk 1]
```diff
@@ -15,25 +15,27 @@ It stores and retrieves arbitrary unstructured data objects via an HTTP-based AP
...
+bucket/object ACL                | partially*                    | partially*                    |
+bucket policies                  | no                            | yes                           |
+object versions                  | no                            | yes                           |
+object expiration                | no                            | yes                           |
+encryption                       | no                            | yes                           |
+CORS                             | no                            | yes                           |
+bucket/IAM user ACP              | no                            | no                            |
+regions and storage classes      | no                            | no                            |
+access logging                   | no                            | no                            |
+website hosting configuration    | no                            | no                            |
```
Comment (matthiasbernhardt) [inline user/pages/04.Reference/05.object-storage/docs.en.md:38]: CORS is part of this, we should group them and clarify what exactly, if not everything from this category, is supported.

File: user/pages/04.Reference/05.object-storage/docs.en.md [hunk 2]
```diff
@@ -15,25 +15,29 @@ It stores and retrieves arbitrary unstructured data objects via an HTTP-based AP
...
+bucket policies                  | no                            | yes                           |
+object versions                  | no                            | yes                           |
+object expiration                | no                            | yes                           |
+encryption                       | no                            | yes                           |
+website hosting configuration    | no                            | no                            |
+CORS                             | no                            | yes                           |
+bucket/IAM user ACP              | no                            | no                            |
+regions and storage classes      | no                            | no                            |
+access logging                   | no                            | no                            |
+
```
Comment (bpetermannS11) [inline user/pages/04.Reference/05.object-storage/docs.en.md:39]: too many blank lines

File: user/pages/04.Reference/05.object-storage/docs.en.md [hunk 3]
```diff
@@ -56,11 +60,13 @@ A file or file object can be assigned a file name like key, and made available u
...
-CBK      | s3.cbk.cloud.syseleven.net  | Yes                 |
-DBL      | s3.dbl.cloud.syseleven.net  | Yes                 |
-FES      | s3.fes.cloud.syseleven.net  | Yes                 |
+Region   | URL                                    | Backend             |
+---------|----------------------------------------|---------------------|
+CBK      | s3.cbk.cloud.syseleven.net             | Quobyte             |
+DBL      | s3.dbl.cloud.syseleven.net             | Quobyte             |
+FES      | s3.fes.cloud.syseleven.net             | Quobyte             |
+FES      | objectstorage.fes.cloud.syseleven.net  | Ceph                |
+
```
Comment (bpetermannS11) [inline user/pages/04.Reference/05.object-storage/docs.en.md:69]: too many blank lines

File: user/pages/04.Reference/04.block-storage/docs.en.md [hunk 4]
```diff
@@ -21,6 +21,8 @@ You can manage your block storage volumes and make them available to your comput
 | ----------------------------------------|---------------|---------------|-----------
 | quobyte                                 | Yes (default) | Yes (default) | Yes (default)
 | quobyte-multiattach                     | Yes           | Yes           | Yes
+| ceph                                    | No            | No            | Yes
+
```
Comment (bpetermannS11) [inline user/pages/04.Reference/04.block-storage/docs.en.md:25]: too many blank lines

**Review by bpetermannS11 — CHANGES_REQUESTED** [review]

File: user/pages/04.Reference/05.object-storage/docs.en.md [hunk 5]
```diff
@@ -15,25 +15,29 @@ It stores and retrieves arbitrary unstructured data objects via an HTTP-based AP
...
+Feature                          | Support in Quobyte            | Support in Ceph               |
+---------------------------------|-------------------------------|-------------------------------|
+bucket HEAD/GET/PUT/DELETE       | yes                           | yes                           |
+object HEAD/GET/PUT/POST/DELETE  | yes                           | yes                           |
+multipart upload                 | yes                           | yes                           |
+bucket listing                   | yes                           | yes                           |
+input compression                | yes                           | yes                           |
+request signature authentication | yes                           | yes                           |
+predefined ACL groups            | yes                           | yes                           |
+bucket/object ACL                | partially*                    | partially*                    |
```
Comment (bpetermannS11) [inline user/pages/04.Reference/05.object-storage/docs.en.md:29]: The (*) doesn't appear below the table

**Review by bpetermannS11 — CHANGES_REQUESTED** [review]

File: user/pages/04.Reference/04.block-storage/docs.en.md [hunk 6]
```diff
@@ -38,6 +39,12 @@ Please refer to our multi-attach volume tutorial. It explains [how to use multi-
 
 !! WARNING: This mode of operation requires special cluster file systems like ocfs2 or gfs2. Otherwise it can lead to the loss of data and/or file system and data corruption.
 
+### ceph
+
+The data will be stored on SSDs in the SysEleven Stack distributed storage cluster based on Ceph. The data will be replicated and stored on three different storage nodes and SSDs.
+
+The volumes can only be attached to a single virtual machine at a time.
```
Comment (bpetermannS11) [inline user/pages/04.Reference/04.block-storage/docs.en.md:46]:
```suggestion
A volume can only be attached to a single virtual machine at a time.
```

**Review by bpetermannS11 — CHANGES_REQUESTED** [review]

File: user/pages/04.Reference/04.block-storage/docs.en.md [hunk 7]
```diff
@@ -38,6 +39,12 @@ Please refer to our multi-attach volume tutorial. It explains [how to use multi-
 
 !! WARNING: This mode of operation requires special cluster file systems like ocfs2 or gfs2. Otherwise it can lead to the loss of data and/or file system and data corruption.
 
+### ceph
+
+The data will be stored on SSDs in the SysEleven Stack distributed storage cluster based on Ceph. The data will be replicated and stored on three different storage nodes and SSDs.
+
+A volume can only be attached to a single virtual machine at a time.
```
Comment (matthiasbernhardt) [inline user/pages/04.Reference/04.block-storage/docs.en.md:46]: Keep in sync with L29/30

File: user/pages/04.Reference/05.object-storage/docs.en.md [hunk 8]
```diff
@@ -15,30 +15,33 @@ It stores and retrieves arbitrary unstructured data objects via an HTTP-based AP
...
+bucket policies                  | no                            | yes                           |
+object versions                  | no                            | yes                           |
+object expiration                | no                            | yes                           |
+encryption                       | no                            | yes                           |
+website hosting configuration    | no                            | no                            |
+CORS                             | no                            | yes                           |
```
Comment (matthiasbernhardt) [inline user/pages/04.Reference/05.object-storage/docs.en.md:35]: shall we write a bit more verbosely "- setting CORS headers" here?

File: user/pages/04.Reference/04.block-storage/docs.en.md (hunk 7 above)
Comment (lgrebe-sys) [reply user/pages/04.Reference/04.block-storage/docs.en.md:46]: Lines are in sync again =)

**Review by cornuwel — DISMISSED** [review]

**Review by MortezaBashsiz — APPROVED** [review]
I have reviewed and seems good from my point of view

**Review by MortezaBashsiz — APPROVED** [review]
Its fine from my side
Approved

**Review by matthiasbernhardt — APPROVED** [review]

**Review by bpetermannS11 — APPROVED** [review]

**Pull request merged**